src/ui/widgets/widgetActions.ts

```diff
@@ -25,7 +26,10 @@ export interface DynamicAction {
     | typeof OPEN_PAGE
     | typeof CLOSE_PAGE
     | typeof OPEN_TAB
-    | typeof CLOSE_TAB;
+    | typeof CLOSE_TAB
+    // TODO: : OPEN_DISPLAY is the keyword used in .opi files, but operates like
+    // OPEN_TAB in cs-web-proto, probably a more efficient way to combine them both
+    | typeof OPEN_DISPLAY;
```

Comment:
Here I'm unsure if one of the existing DynamicAction properties is what I need to use, currently I copied the action for `OPEN_TAB` as I didn't want to tie the `OPEN_DISPLAY` (the property I made) to something like open tab that may be changed in the future. Thoughts would be good.